api/sql/schema/750-rule.rule.remove.sql

```diff
@@ -6,64 +6,6 @@ DECLARE @userId BIGINT = (SELECT [auth.actorId] FROM @meta)
 BEGIN TRY
 
     BEGIN TRANSACTION
-        DELETE x
-        FROM
-            [rule].limit x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE x
-        FROM
-            [rule].conditionActor x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE x
-        FROM
-            [rule].conditionItem x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE x
-        FROM
-            [rule].conditionProperty x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE
-            x
-        FROM
-            [rule].splitRange x
-        JOIN
-            [rule].splitName s ON s.splitNameId = x.splitNameId
-        JOIN
-            @conditionId item ON s.conditionId = item.value
-        DELETE
-            x
-        FROM
-            [rule].splitAnalytic x
-        JOIN
-            [rule].splitAssignment y ON y.splitAssignmentId = x.splitAssignmentId
-        JOIN
-            [rule].splitName s ON s.splitNameId = y.splitNameId
-        JOIN
-            @conditionId item ON s.conditionId = item.value
-
-        DELETE
-            x
-        FROM
-            [rule].splitAssignment x
-        JOIN
-            [rule].splitName s ON s.splitNameId = x.splitNameId
-        JOIN
-            @conditionId item ON s.conditionId = item.value
-        DELETE
-            x
-        FROM
-            [rule].splitName x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
         UPDATE x
         SET isDeleted = 1,
```

Comment:
Should we change the name? We must if we want later to add a new rule with this name? If we want to keep the name always unique no matter deleted or not then it is ok

Reply:
Let's leave this for another MR, that also include appropriate tests for deletion.